compiler-rt/test/sanitizer_common/TestCases/corelimit.cpp

```diff
@@ -10,7 +10,12 @@ int main() {
   getrlimit(RLIMIT_CORE, &lim_core);
   void *p;
   if (sizeof(p) == 8) {
-    assert(0 == lim_core.rlim_cur);
+#ifdef __linux__
+    // See comments in DisableCoreDumperIfNecessary().
+    assert(lim_core.rlim_cur == 1);
+#else
+    assert(lim_core.rlim_cur == 0);
```

Comment:
`assert(lim_core.rlim_cur == SANITIZER_LINUX);` or somilar

Reply:
We don't have that define available in this standalone test binary, so I've kept the ifdefs.

Reply:
off-cause, it's a test
sorry